resources/server/src/landingpage/README.md

```diff
@@ -1,28 +1,37 @@
-# Landing page/catalog
+# Landing page/catalog webapp
 
-Landing page/catalog app source code.
+[Landing page/catalog webapp](https://docs.qgis.org/latest/en/docs/server_manual/services.html#qgis-server-catalog) source code.
 
-## Project setup
-```
-yarn install
-```
+## Building
 
-### Compiles and hot-reloads for development
-```
-yarn serve
-```
+To build the QGIS server landingpage webapp additional dependencies are required:
+
+Node.js (current LTS recommended): https://nodejs.org/en/download/<br>
+Yarn Package Manager: https://yarnpkg.com/getting-started/install
+
+To build the webapp along with QGIS server just turn the cmake flag `WITH_SERVER_LANDINGPAGE_WEBAPP` on and build as you would normally do.
+
+To test the webapp with a local QGIS project after your build is finished, set the [environment variable](https://docs.qgis.org/latest/en/docs/server_manual/config.html#environment-variables) for your project directory and run the [development server](https://docs.qgis.org/latest/en/docs/server_manual/development_server.html):
 
-### Compiles and minifies for production
 ```
-yarn build
+export QGIS_SERVER_LANDING_PAGE_PROJECTS_DIRECTORIES=/path/to/projectdirectory
+
+./output/bin/qgis_mapserver -p /path/to/projectdirectory/test.qgz
 ```
 
-Built files location: `../../api/ogc/static/landingpage/`
+Then open the web browser on http://localhost:8000 (for default port `8000`).
+
+
+## Development
+
+Development could be done by modifying the source code and building and running the webapp as described above.
+
+If you want to run `yarn install` manually, e.g. to update `yarn.lock` please make sure to remove the `node_modules` directory from your QGIS source tree before running cmake and building. The same accounts for removing the `landingpage` directory from `resources/server/api/ogc/static/` after running `yarn build` manually.
```

Comment:
Can you reformulate these two paragraphs to be more explicit (i.e. it should be clear for someone external how to update the `yarn.lock` file). I.e. change the wording that is currently "could" and "want" with something more explitit.

Reply:
Thanks for reviewing and your suggestions :+1: , done in https://github.com/qgis/QGIS/pull/46791/commits/b4f5668e504481d49a36f9104a3451c86a360d6a